src/assessments/custom-widgets/test-steps/design-pattern.tsx

```diff
@@ -26,7 +26,12 @@ const designPatternDescription: JSX.Element = (
 
 const designPatternHowToTest: JSX.Element = (
     <div>
-        For this requirement, {productName} highlights custom widgets. (A custom widget is an element with a valid ARIA widget role.)
+        <p>For this requirement, {productName} highlights custom widgets. (A custom widget is an element with a valid ARIA widget role.)</p>
+        <p>
+            <Markup.Emphasis>
```

Comment:
looks to me that this can be extracted and re-use throughout the content

Reply:
Accepted.